certbot-apache/certbot_apache/constants.py

```diff
@@ -62,9 +62,9 @@
     vhost_files="*.conf",
     logs_root="/var/log/apache2",
     version_cmd=['/usr/sbin/apache2', '-v'],
-    define_cmd=['apache2ctl', 'virtualhosts'],
-    restart_cmd=['apache2ctl', 'graceful'],
-    conftest_cmd=['apache2ctl', 'configtest'],
+    define_cmd=['/usr/sbin/apache2', '-S'],
```

Comment:
In contrast to other distributions Gentoo does not install any "default" by default. The user always has to choose what he uses and not if he wants to exchange a default system with something different.

As I say, it currently is completely useless for all systemd users while only a few OpenRC users make use of the auto installation feature (especially on Gentoo most will very likely manage their vhosts on their own anyway). Currently there is no way on systemd installations to use certbot, while with this change at least everyone is able to use the certonly option.